### PR TITLE
add debian 11 compatibility

### PR DESCRIPTION
**Pull request by kwypchlo**
still draft

### CODE REVIEW

File: playbooks/all-docker-install.yml [hunk 1]
```diff
@@ -17,31 +17,26 @@
...
       run_once: True
 
-    # Check host OS version
-    - name: Check supported OS and version
-      assert:
-        that:
-          - ansible_distribution|lower == 'debian' and ansible_distribution_version == '10'
-        fail_msg: "{{ ansible_distribution }} {{ ansible_distribution_version }} is not yet supported by this role"
-    
+    - name: Check server OS compatibility
```
Comment (firyx) [inline playbooks/all-docker-install.yml:25]: Convention: `- name: Include checking ...`

File: playbooks/deploy-machines-setup.yml [hunk 2]
```diff
@@ -10,27 +10,21 @@
...
     - role: geerlingguy.docker
       become: True
 
     # Secure the server
     - role: geerlingguy.security
       become: True
-  
+
   tasks:
+    - name: Check server OS compatibility
```
Comment (firyx) [inline playbooks/deploy-machines-setup.yml:25]: same: `Include checking ...`

File: playbooks/portals-setup-following.yml [hunk 3]
```diff
@@ -13,11 +13,8 @@
     - name: Check '--limit' is used
       include_tasks: tasks/host-limit-check.yml
 
-    - name: Check supported OS and version
-      assert:
-        that:
-          - ansible_distribution|lower == 'debian' and ansible_distribution_version == '10'
-        fail_msg: "{{ ansible_distribution }} {{ ansible_distribution_version }} is not yet supported by this role"
+    - name: Check server OS compatibility
```
Comment (firyx) [inline playbooks/portals-setup-following.yml:16]: same

File: playbooks/portals-setup-initial.yml [hunk 4]
```diff
@@ -38,25 +38,22 @@
...
       ansible.builtin.setup:
-    
-    - name: Check supported OS and version
-      assert:
-        that:
-          - ansible_distribution|lower == 'debian' and ansible_distribution_version == '10'
-        fail_msg: "{{ ansible_distribution }} {{ ansible_distribution_version }} is not yet supported by this role"
-    
+
+    - name: Check server OS compatibility
```
Comment (firyx) [inline playbooks/portals-setup-initial.yml:54]: same

**Review by firyx — CHANGES_REQUESTED** [review]
Please add changelog item.

File: playbooks/portals-setup-initial.yml (hunk 4 above)
Comment (kwypchlo) [reply playbooks/portals-setup-initial.yml]: renamed in https://github.com/SkynetLabs/ansible-playbooks/pull/238/commits/de6d344272248acf2313779c0dbc53758035699c

File: playbooks/portals-setup-following.yml (hunk 3 above)
Comment (kwypchlo) [reply playbooks/portals-setup-following.yml]: renamed in https://github.com/SkynetLabs/ansible-playbooks/pull/238/commits/de6d344272248acf2313779c0dbc53758035699c

File: playbooks/deploy-machines-setup.yml (hunk 2 above)
Comment (kwypchlo) [reply playbooks/deploy-machines-setup.yml]: renamed in https://github.com/SkynetLabs/ansible-playbooks/pull/238/commits/de6d344272248acf2313779c0dbc53758035699c

File: playbooks/all-docker-install.yml (hunk 1 above)
Comment (kwypchlo) [reply playbooks/all-docker-install.yml]: renamed in https://github.com/SkynetLabs/ansible-playbooks/pull/238/commits/de6d344272248acf2313779c0dbc53758035699c

**Comment by kwypchlo** [comment]
ready for review, init and deploy worked peachy on new servers with debian 11

**Comment by kwypchlo** [comment]
debian 11 support added here too https://github.com/SkynetLabs/ansible-role-skynet-webportal/pull/1

**Review by ro-tex — APPROVED** [review]

**Review by firyx — APPROVED** [review]

**Pull request merged**